evoting/lib/transaction.go

```diff
@@ -187,9 +187,12 @@ func (t *Transaction) Verify(genesis skipchain.SkipBlockID, s *skipchain.Service
 		}
 		return nil
 	} else if t.Ballot != nil {
-		election, err := GetElection(s, genesis, false, t.User)
-		if err != nil {
-			return err
+		zero := cothority.Suite.Point()
```

Comment:
According to this: https://github.com/dedis/kyber/blob/master/group.go#L145, you should compare to `cothority.Suite.Point().Null()`